src/drivers/camera_trigger/camera_trigger.cpp

```diff
@@ -876,7 +881,12 @@ int camera_trigger_main(int argc, char *argv[])
 			return 1;
 		}
 
-		camera_trigger::g_camera_trigger->start();
+		if (!camera_trigger::g_camera_trigger->start()) {
+			camera_trigger::g_camera_trigger->stop();
```

Comment:
If `start` fails, I would not expect to have to call `stop`. It's not needed here.